cookbooks/bach_spark/attributes/default.rb

```diff
@@ -1,60 +1,68 @@
-user = node['bcpc']['bootstrap']['admin']['user']
-
-default[:spark][:download][:url] = 'http://d3kbcqa49mib13.cloudfront.net'
-default[:spark][:download][:file][:name] = 'spark-2.1.0-bin-hadoop2.7'
-default[:spark][:download][:file][:type] = 'tgz'
-default[:spark][:download][:dir] = "/home/#{user}/chef-bcpc/bins"
-default[:spark][:package][:install_meta] = false
-default[:spark][:package][:base] = '/usr/spark'
-default[:spark][:package][:prefix] = 'spark'
-default[:spark][:package][:version] = '2.1.0'
 default[:spark][:hdfs_url] = node['bcpc']['hadoop']['hdfs_url']
-default[:spark][:bin][:dir] = "#{node[:spark][:package][:base]}/"\
```

Comment:
Should probably leave this attribute here, but just make the value `'/usr/hdp/current/spark2-client'`